### PR TITLE
Add include "additional XML files" functionality

### PR DESCRIPTION
**Pull request by photodude**
[As per reference in tutorial on Custom XML form field types](https://docs.joomla.org/Creating_a_custom_form_field_type)
Adding XML include file functionality
XML form definitions are cumbersome, and maintaining a number of similar forms can be problematic if fieldset and field definitions have only minor differences between forms.

You can alleviate a lot of the drudgery by grouping your fieldset or field definitions in additional files and then include those files in your top level form definition.

This adds a great deal of flexibility and a reduction in developer duplication of effort.

### CODE REVIEW

**Comment by mbabker** [comment]
Can you send this to the https://github.com/joomla-framework/form repo?  This one's effectively shut down.

**Comment by photodude** [comment]
As requested by @mbabker , this has been created on the  https://github.com/joomla-framework/form repo as PR22

**Comment by photodude** [comment]
Since this Repo is "effectively shut down"  and the PR in now at https://github.com/joomla-framework/form/pull/22 I'm closing the PR at this Repo

File: src/Joomla/Form/Form.php [hunk 1]
```diff
@@ -766,8 +766,8 @@ public function loadFile($file, $reset = true, $xpath = false)
 			}
 		}
 
-		// Attempt to load the XML file.
-		$xml = simplexml_load_file($file);
+		// Attempt to load the XML files.
+		$xml = simplexml_load_file($file, 'SimpleXMLElement', LIBXML_NOENT);
```
Comment (izharaazmi) [inline src/Joomla/Form/Form.php:770]: I understand that this is no place to discuss things, but I could not think of any appropriate place for clarification of my doubt.
@photodude As far as I understand `LIBXML_NOENT` makes simple xml to substitute entities in the XML. Can you explain how that is significant with "Adding XML include file functionality"?

**Comment by photodude** [comment]
@izharaazmi  please refer to the current open PR for this issue. https://github.com/joomla-framework/form/pull/22

I have the same understanding that `LIBXML_NOENT` makes simple xml to substitute entities in the XML. It accomplishes this by a referenced file(s) in the XML file loaded by `simplexml_load_file` hence "Adding XML include file functionality" I'll clairify that by specifying that it's including additional XML file(s)

**Pull request closed without merging**